README.md

```diff
@@ -46,7 +46,7 @@ const datadog = new Datadog(this, "Datadog", {
   nodeLayerVersion: <LAYER_VERSION>,
   pythonLayerVersion: <LAYER_VERSION>,
   addLayers: <BOOLEAN>,
-  forwarderArn: "<FORWARDER_ARN>",
```

Comment:
Let's not delete this one, since it's still supported. I think this sample code tries to list out all the possible parameters.